README.md

```diff
@@ -51,6 +51,42 @@ Any ZeroMQ subscriber can listen to those events and act acordingly.
 
 > ```sudo systemctl enable ${PROJECT_NAME}.service && sudo systemctl start ${PROJECT_NAME}.service```
 
+## Docker
+
+To build the docker image, from the clonned repository, execute the docker build command in the same level as the Dockerfile.
+
+```docker build -t ${IMAGE_NAME}:${IMAGE_VERSION} .```
+
+For example:
+```docker build -t iombian-button-handler:latest .```
+
+After building the image, execute it with docker run
+
+```docker run --network host -e BUTTON_PIN=3 --privileged --rm -d --name ${CONTAINER_NAME} ${IMAGE_NAME}```
```

Comment:
```docker run --name ${CONTAINER_NAME} --privileged --rm -d -p 5556:5556 -e BUTTON_PIN=3 ${IMAGE_NAME}:${IMAGE_VERSION}```

I would prefer if the official documentation did not recommend the use of "--network host" and suggested the mapping of the port to use: 5556.